src/utils/safeTranslate.ts

```diff
@@ -0,0 +1,24 @@
+import { TFunction, TFunctionDetailedResult } from "i18next";
+import { logger } from "./logger";
+import { LOCALE } from "../app.constants";
+
+export type translateResult = string | TFunctionDetailedResult<string, any>;
+
+export const safeTranslate = (
+  translate: TFunction<"translation", undefined>,
+  key: string,
+  options?: Record<string, any>
+): translateResult => {
+  const result: translateResult = translate(key, options);
+  if (result === key) {
+    logger.error(
+      `TranslationError: key '${key}' missing for requested language.`
```

Comment:
It's not clear what's in the options object - do we have easy access to which language we're looking for? It'd be helpful to know if the missing key is English or Welsh. 

It's not going to be hard to search for the key in two files, but if we can easily put it in the logs we should

Reply:
Done